README.md

```
@@ -1,88 +1,95 @@
+```css
+:root {
+    --primary-color: #6366f1;
+    --secondary-color: #8b5cf6;
+    --accent-color: #06b6d4;
+    /* ... more variables */
+}
+```
+
+## 📁 Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)